docs/js-core-3/week-2/lesson.md

```diff
@@ -425,37 +425,95 @@ fetch(_____)
+
+However, it is important not to overuse positive feedback as its value will decrease. You may find it more valuable to hear positive comments from someone who almost never praises you for your work than a coworker constantly telling you how awesome you are. Too much positive feedback can also make employees complacent and feel less challenged in their roles.
+
+Also, specific positive feedback can be much more effective than general positive feedback. The first feedback is better than the second one: 
+1. The way you presented graphs to show our progress really made it easy to notice what tasks I needed to focus on which helped me get the project delivered.
+2. It was great working with you, it helped the project succeed.
+
+**Negative feedback:** This is the kind of feedback that people don’t like to hear, especially without warning. Negative feedback can make individuals feel attacked, demotivated, and undervalued at work.
+
+However, negative feedback can be effective when utilized correctly. It becomes **constructive feedback** and identifies ways an individual can improve.
+
+**Destructive feedback** is the direct opposite of constructive feedback and it’s not very useful. Instead of focusing on the work, destructive feedback will focus on the individual and is very personal in nature. There is little productive advice given and the feedback tends to point at faults without any solutions.
+
+:::note Exercise (25 min)
+
+**Exercise objective:** To practice providing both positive and negative feedback.
+
+Work in small groups. Take turns to play the roles below and answer the following questions for each situation:
+
+- Was feedback positive or negative?
+- Was the way it was communicated appropriate? Why?
+- How would you improve it?
+- What would be your reaction after receiving such feedback?
+- Were people in the right mindset to give and receive feedback?
+
+*Situation 1*: Tom asked Katie for help on the code he was writing. 
 
-[Agile: Planning and preparation](https://docs.google.com/presentation/d/1TJNjPOha2P5cWW6IO9jAc9GTvsosqVGkkG5pAbbvfg4/edit#slide=id.g1b740a9f8c6_0_336)
+Tom: “Hi Katie, would you mind spending a few minutes looking at this function I am working on? I can’t seem to find what’s wrong with the code.”
 
-:::note Exercise (30 minutes)
+Katie: “I am extremely busy and stressed with a deadline, but let me have a look.”
+
+Tom: “Here is the function. Do you see anything wrong with it?”
+
+Katie: After looking at the code for some time… “Seriously? How could you miss this? You are missing an End statement. You need to be more careful when writing code so you only ask for help for things that are not this obvious.”
+
+Katie walks away.
+
+*Situation 2*: Patricia stops by Carla’s desk.
+
+Patricia: “Hi Carla. Just wanted to let you know how much I appreciate the work you did in the last project and I hope we get to work together again”
```

Comment:
This feels like fairly general positive feedback. Perhaps we could make it more specific?

```suggestion
Patricia: “Hi Carla. Just wanted to let you know how much I appreciate your clear messages on Slack you did in the last project, they really helped me understand what you were working on. I hope we get to work together again”
```